### PR TITLE
Use `id_token` claim for getUserAttributes() if 'userinfo_endpoint' is not available.

### PR DESCRIPTION
**Pull request by rhertogh**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | #240

### CODE REVIEW

File: src/OpenIdConnect.php [hunk 1]
```diff
@@ -75,6 +76,20 @@
  */
 class OpenIdConnect extends OAuth2
 {
+    // https://openid.net/specs/openid-connect-core-1_0.html#rfc.section.2
```
Comment (samdark) [inline src/OpenIdConnect.php:79]: That should be in a docblock that also contains `@since 2.2.12`.

File: src/OpenIdConnect.php [hunk 2]
```diff
@@ -233,6 +248,16 @@ protected function discoverConfig()
         return $response;
     }
 
+    /**
+     * Set the OpenID provider configuration manually, this will bypass the automatic discovery via
+     * the /.well-known/openid-configuration endpoint.
+     * @param array $configParams OpenID provider configuration parameters.
```
Comment (samdark) [inline src/OpenIdConnect.php:254]: `@since 2.2.12`

**Review by samdark — CHANGES_REQUESTED** [review]

**Comment by samdark** [comment]
Overall it looks 👍

File: src/OpenIdConnect.php (hunk 1 above)
Comment (rhertogh) [reply src/OpenIdConnect.php]: Updated docblock

File: src/OpenIdConnect.php (hunk 2 above)
Comment (rhertogh) [reply src/OpenIdConnect.php:258]: added

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**